templates/superset_config.py

```diff
@@ -31,7 +32,83 @@
     port=os.getenv("REDIS_PORT"),
     key_prefix="superset_results",
 )
+TALISMAN_ENABLED = True
+TALISMAN_CONFIG = {
+    "content_security_policy": {
+        "default-src": ["'self'"],
+        "img-src": ["'self'", "data:"],
+        "worker-src": ["'self'", "blob:"],
+        "connect-src": [
+            "'self'",
+            "https://api.mapbox.com",
+            "https://events.mapbox.com",
+        ],
+        "object-src": "'none'",
+        "style-src": ["'self'", "'unsafe-inline'"],
+        "script-src": ["'self'", "'unsafe-inline'"],
+    },
+    "force_https": False,
+}
+
+SQLALCHEMY_POOL_SIZE = 5  # Maximum number of connections in the pool
```

Comment:
Is there a need for these to be configurable?

Reply:
Yes likely, I will include this in my next PR which makes the `FEATURE_FLAGS` and `beat_schedule` configurable.